independent-publisher-connectors/XSOAR/apiDefinition.swagger.json

```diff
@@ -0,0 +1,100 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Provides the ability to submit content to XSOAR integrations when using instance.execute.external to expose integration instances over HTTPS. Tested to work with the Generic Webhook integration.",
+    "version": "1.0.0",
+    "title": "XSOAR",
+    "contact": {
+      "name": "Landon Chelf",
+      "email": "landon.chelf@fisglobal.com"
+      "url": "https://fisglobal.com"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "host": "demisto.live",
+  "basePath": "/instance/execute",
+  "schemes": [
+    "https"
+  ],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.paloaltonetworks.com/"
```

Comment:
@landonc Please provide specific website URL of the connector service like below:
https://start.paloaltonetworks.com/sign-up-for-community-edition.html